src/main/java/pro/beam/api/resource/chat/events/data/IncomingMessageData.java

```diff
@@ -17,6 +17,7 @@
     @SerializedName("user_id") public int userId;
     @SerializedName("user_roles") public List<BeamUser.Role> userRoles;
     public MessageComponent message;
+    public String target;
```

Comment:
It's not. In the current version of the whisper system, you can assume that the target will always be the currently authenticated user. In future iterations, if it does still use the message system, the target user will probably be some property of the meta.